src/main/java/com/endava/webhw/controller/DepartmentController.java

```diff
@@ -0,0 +1,56 @@
+package com.endava.webhw.controller;
+
+import com.endava.webhw.dto.DepartmentDto;
+import com.endava.webhw.model.Department;
+import com.endava.webhw.service.DepartmentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Wildcard import is a bad practice

Reply:
Removed wildcard from all classes where it was found and all tests as well.